### ISSUE TITLE
Account confirmation

### ISSUE THREAD

**Issue by tarkal** [ff36/jnoc#72]
Currently if a user does not authorize their account before the activation link expires they cannot confirm their account. There should be a button in the admin console that allows admin to either manually confirm account or resend a new activation link.

**Comment by cantorfan** [comment]
I tested and don't find a token expired date,  please reivew and help to confirm.

**Comment by tarkal** [comment]
You are correct there is no `token expired date`. This is because it is dynamically calculated at runtime from the creation date. The system periodically scans the `Token` table and finds all expired tokens to prevent the Table from filling up indefinitely with tokens and removes them. 

N.B. Regardless of this scan it is possible that a user deletes or does not receive the original email so we need a mechanism to allow a new confirmation token to be sent out.
